docs/troubleshooting/README.md

```diff
@@ -1 +1,717 @@
+
+We need to have `ClusterResourcePlacement` name `{CRPName}`, replace `{CRPName}` in the command below,
+
+```
+kubectl get clusterschedulingpolicysnapshot -l kubernetes-fleet.io/is-latest-snapshot=true,kubernetes-fleet.io/parent-CRP={CRPName}
+```
+
+- Compare `ClusterSchedulingPolicySnapshot` with the `ClusterResourcePlacement` policy to ensure they match (excluding `numberOfClusters` field from `ClusterResourcePlacement` spec).
+- If placement type is `PickN`, check if the number of clusters requested in `ClusterResourcePlacment` placement policy matches the value for the label called `number-of-clusters`.
+
+## How to find the latest ClusterResourceBinding resource?
+
+We need to have `ClusterResourcePlacement` name `{CRPName}`, replace `{CRPName} `in the command below. The command below lists all `ClusterResourceBindings` associated with `ClusterResourcePlacement`,
+
+```
+kubectl get clusterresourcebinding -l kubernetes-fleet.io/parent-CRP={CRPName}
+```
+
+### Example:
+
+In this case we have `ClusterResourcePlacement` called test-crp,
+
+```
+kubectl get crp test-crp
+NAME       GEN   SCHEDULED   SCHEDULEDGEN   APPLIED   APPLIEDGEN   AGE
+test-crp   1     True        1              True      1            15s
+```
+
+From the `placementStatuses` section of the `test-crp` status, we can observe that it has propagated resources to two member clusters and hence has two `ClusterResourceBindings`,
+
+```
+status:
+  conditions:
+  - lastTransitionTime: "2023-11-23T00:49:29Z"
+    ...
+  placementStatuses:
+  - clusterName: kind-cluster-1
+    conditions:
+      ...
+      type: ResourceApplied
+  - clusterName: kind-cluster-2
+    conditions:
+      ...
+      reason: ApplySucceeded
+      status: "True"
+      type: ResourceApplied
+```
+
+Output we receive after running the command listed above to get the `ClusterResourceBindings`,
+
+```
+kubectl get clusterresourcebinding -l kubernetes-fleet.io/parent-CRP=test-crp 
+NAME                               WORKCREATED   RESOURCESAPPLIED   AGE
+test-crp-kind-cluster-1-be990c3e   True          True               33s
+test-crp-kind-cluster-2-ec4d953c   True          True               33s
+```
+
+The `ClusterResourceBinding` name follows this format `{CRPName}-{clusterName}-{suffix}`, so once we have all ClusterResourceBindings listed find the `ClusterResourceBinding` for the target cluster you are looking for based on the `clusterName`.
+
+## How to find the latest ClusterResourceSnapshot resource?
+
+Replace `{CRPName}` in the command below with name of `ClusterResourcePlacement`,
+
+```
+kubectl get clusterresourcesnapshot -l kubernetes-fleet.io/is-latest-snapshot=true,kubernetes-fleet.io/parent-CRP={CRPName} -o YAML
+```
+
+## How and where to find the correct Work resource?
+
+We need to have the member cluster's namespace which follow this format `fleet-member-{clusterName}`, `ClusterResourceBinding` name `{CRBName}` and `ClusterResourcePlacement` name `{CRPName}`.
```

Comment:
we don't need the `ClusterResourceBinding` name right? we can still get the work created for the CRP.